tests/data/parsers/test_parser_classification_rule.py

```diff
@@ -220,19 +224,51 @@ def test_feedback_for_criteria_with_gaps_and_overlap(
                 [2, "<0", "<0", ">10"],
                 [3, "-", "-", "-"],
             ],
-            """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0). Gap for variable varB in range 0.0:inf\nOverlap for variable varA in range -inf:0.0""",
+            [
+                call(
+                    """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0). Gap for variable varB in range 0.0:inf\nOverlap for variable varA in range -inf:0.0"""
+                )
+            ],
+        ),
+        (
+            [
+                ["output", "MIN_water_depth_mNAP", "MAX_flow_velocity", "MAX_chloride"],
+                [1, "<0.10", "-", ">300"],  # to dry
+                [2, "<0.10", "-", "< 400"],  # also to dry
+                [3, ">4.0", ">3.0", "-"],  # to deep
+                [4, ">4.0", "<2.0", "-"],  # also to deep
+                [5, "-", "-", ">400"],  # to salty
+                [6, "-", ">1.5", "-"],  # to fast flowing
+                [7, "-", ">1.5", ">300"],  # alos to fast flowing
+                [8, "0.20:4.0", "0.0:1.5", "0:400"],  # perfect for aquatic plants
+            ],
+            [
+                call(
+                    "For conditions: (MIN_water_depth_mNAP: -, MAX_flow_velocity: -). Gap for variable MAX_chloride in range -inf:400.0\nFor conditions: (MIN_water_depth_mNAP: -, MAX_flow_velocity: >1.5). Overlap for variable MAX_chloride in range 300.0:inf\nFor conditions: (MIN_water_depth_mNAP: 0.20:4.0, MAX_flow_velocity: 0.0:1.5). Gap for variable MAX_chloride in range -inf:0.0\nFor conditions: (MIN_water_depth_mNAP: 0.20:4.0, MAX_flow_velocity: 0.0:1.5). Gap for variable MAX_chloride in range 400.0:inf\nFor conditions: (MIN_water_depth_mNAP: <0.10, MAX_flow_velocity: -). Overlap for variable MAX_chloride in range 300.0:400.0\nFor conditions: (MIN_water_depth_mNAP: -). Overlap for variable MAX_flow_velocity in range 1.5:inf"
+                ),
+                call(
+                    "12 warnings found concerning coverage of the parameters. Only first 6 "
+                    "warnings are shown. See classification_warnings.log file for all warnings."
```

Comment:
see the classification_warnings.log file for...
or see classification_warnings.log for...
also this file is only for the multiple_classification rule -> correct?
Then that should be the name: "multiple_classification_rule_warnings.log"

Reply:
the warnings come from the parser (actually the validation). Is multiple_classification_warnings.log ok?

Reply:
but shouldn't it point the user to which rule it comes from?
multiple_classification_warnings.log is fine, but perhaps not clear enough?